app/experimenter/templates/projects/edit.html

```diff
@@ -0,0 +1,63 @@
+{% extends "base.html" %}
+
+{% load static %}
+
+{% block content %}
+  <div class="row">
+    <div class="col-md-8 col-md-offset-2 card">
+      <form 
+        class="form-horizontal card-content" 
+        method="POST" 
+        action=""
```

Comment:
According to the [HTML5 spec](http://w3c.github.io/html/sec-forms.html#form-control-infrastructure-form-submission) it's better to leave the action out entirely if the form action is the document's URL.